app/controllers/host_genomes_controller.rb

```diff
@@ -27,6 +27,14 @@ def new
   def edit
   end
 
+  # GET /host_genomes/add
```

Comment:
Are these new endpoints intentional? It doesn't look like there is a corresponding view file (e.g. `add.html.erb`) for them.

Reply:
oh I accidentally check that in. It's actually for another diff. I will revert. thanks.